src/components/exercise-generation/use-ai-features.tsx

```diff
@@ -1,11 +1,14 @@
 import { Ai } from '@serlo/authorization'
+import { useRouter } from 'next/router'
 
 import { useCanDo } from '@/auth/use-can-do'
 import { isProduction } from '@/helper/is-production'
 
 export function useAiFeatures() {
   const canDo = useCanDo()
-  const canUseAiFeatures = canDo(Ai.executePrompt)
+  const { locale } = useRouter()
```

Comment:
just a note:
if you use this you get types:
```suggestion
  const { lang } = useInstanceData()
```